sql/src/test/java/io/crate/metadata/information/InformationSchemaIngestionRulesTest.java

```diff
@@ -0,0 +1,94 @@
+package io.crate.metadata.information;
+
+
+import com.google.common.collect.Iterators;
+import io.crate.metadata.IngestionRuleInfo;
+import io.crate.metadata.IngestionRuleInfos;
+import io.crate.metadata.rule.ingest.IngestRule;
+import io.crate.metadata.rule.ingest.IngestRulesMetaData;
+import io.crate.test.integration.CrateDummyClusterServiceUnitTest;
+import io.crate.testing.SQLExecutor;
+import org.elasticsearch.action.admin.cluster.state.ClusterStateRequestBuilder;
+import org.elasticsearch.cluster.ClusterState;
+import org.elasticsearch.cluster.metadata.MetaData;
+import org.elasticsearch.test.ClusterServiceUtils;
+import org.junit.Test;
+
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.StreamSupport;
+
+import static org.hamcrest.Matchers.is;
+
+
+public class InformationSchemaIngestionRulesTest extends CrateDummyClusterServiceUnitTest {
+
+    private IngestionRuleInfos ingestionRuleInfos;
+
+    public void setUpTests(Boolean emptyMetaData) throws Exception {
+        Map<String, Set<IngestRule>> sourceRules = new HashMap<>();
+
+        Set<IngestRule> rules_mqtt = new HashSet<>();
+        rules_mqtt.add(new IngestRule("processV4TopicRule", "mqtt_raw", "topic like v4/%"));
+        rules_mqtt.add(new IngestRule("processV5TopicRule", "mqtt_5_raw", "topic like v5/%"));
+        sourceRules.put("mqtt", rules_mqtt);
+
+        Set<IngestRule> rules_http = new HashSet<>();
+        rules_http.add(new IngestRule("http_rule", "http_raw", "topic like http/%"));
+        sourceRules.put("http", rules_http);
+
+        IngestRulesMetaData inputMetaData = new IngestRulesMetaData(sourceRules);
+
+        ClusterState.Builder builder = ClusterState.builder(clusterService.state());
+
+        if (emptyMetaData == false) {
+            builder.metaData(MetaData.builder(clusterService.state().metaData())
+                    .putCustom(IngestRulesMetaData.TYPE, inputMetaData));
+        } else {
+            builder.metaData(MetaData.builder(clusterService.state().metaData()));
+        }
+        ClusterState clusterState = builder.build();
+        ClusterServiceUtils.setState(clusterService, clusterState);
```

Comment:
pls move both lines up to the `if (emptyMetaData == false) {}` branch and remove the else branch